src/sql/workbench/services/connection/browser/cmsConnectionWidget.ts

```diff
@@ -108,8 +108,9 @@ export class CmsConnectionWidget extends ConnectionWidget {
 			} else {
 				authTypeOption.defaultValue = this.getAuthTypeDisplayName(AuthenticationType.SqlLogin);
 			}
-			this._authTypeSelectBox.setOptions(authTypeOption.categoryValues.map(c => c.displayName), 1);
 		}
+		this._authTypeSelectBox.setOptions(authTypeOption.categoryValues.map(c => c.displayName), 1);
+		this._authTypeSelectBox.selectWithOptionName(authTypeOption.defaultValue);
 	}
```

Comment:
How does this fix the issue described? An error when expanding the node doesn't seem like it should be affected by the default value in the connection dialog...

Reply:
Default values matter when setting up the authorization type

Reply:
But how is that impacting the connection that is made when you expand a node in the tree? That doesn't necessarily involve opening the connection dialog at all.

Reply:
Just realized I had the incorrect issue tagged here. Updated the issue. Sorry for the confusion!